tests/system/llm/test_openai.py

```diff
@@ -73,6 +73,10 @@ def openai_llm(request):
         if os.getenv("ANYSCALE_API_KEY") is None:
             pytest.skip("Couldn't find Anyscale API key. Skipping Anyscale tests.")
         model_name = "mistralai/Mistral-7B-Instruct-v0.1"
+    elif llm_class == OctoAILLM:
+        if os.getenv("OCTOAI_API_KEY") is None:
```

Comment:
@ptorru we have a bug here that blocks the merge can we fix it? (Removing the brackets)